datasets/InputData_Mixed.csv

```diff
@@ -0,0 +1,7 @@
+timestamp,Outside Air Temperature (C),Outside Air Relative Humidity (1-100%),Outside Air Humidity Ratio (-),Zone Thermostat Requested Sensible Heat Transfer Rate (kW),Supply Air Temperature (C),Supply Air Relative Humidity (0-100%),Supply Air Humidity Ratio,Water Use (Liters/min),System Electric Power (kW),Operating Mode,Supply Air Mass Flow Rate (kg/s),Supply Air Volume Flow Rate (m3/s),Sensible System Cooling Capacity (kW),Latent System Cooling Capacity (kW),Total System Cooling Capacity (kW)
```

Comment:
So this is input data for the unit test. It relates to a specific system, and so we can test against the expected results of the real system. For our validation we have a 40K line long file of data that we are using to validate the performance of the algorithm. This little file is just a few time steps testing different modes. Im not sure where else it belongs, but we do need it.